RadixWallet/Features/SettingsFeature/ManualAccountRecoveryScan/ManualAccountRecoverySeedPhraseCoordinator+Reducer.swift

```diff
@@ -0,0 +1,187 @@
+import ComposableArchitecture
+import SwiftUI
+
+// MARK: - ManualAccountRecoverySeedPhraseCoordinator
+public struct ManualAccountRecoverySeedPhraseCoordinator: Sendable, FeatureReducer {
+	public typealias Store = StoreOf<Self>
+
+	// MARK: - State
+
+	public struct State: Sendable, Hashable {
+		public var selected: EntitiesControlledByFactorSource? = nil
```

Comment:
Need value indicating if it is an Olympia or Babylon scan - which will AccountRecoveryScanCoordinator requires - which affects if a CAP26 or BIP44Olymlia derivation path is used.